msf_updates/src/position_msf/init_yaw.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+PACKAGE = 'msf_updates'
+import roslib;roslib.load_manifest(PACKAGE)
+import rospy
+from geometry_msgs.msg import Vector3, Quaternion
+from std_msgs.msg import Float64
+import tf
+from math import *
+
+import dynamic_reconfigure.client
+
+client = dynamic_reconfigure.client.Client(rospy.get_param("msf_sensor_node",
```

Comment:
Thanks! :) Do we need a YAML, or is it also fine to specify the name of the node directly in the launch file? I can make a YML in msf_updates/cfg with the string name if necessary.